int8 face_detected
int32 face_vertical_displacement
int32 face_horizontal_displacement

int8 FACE_DETECED = 0
int8 NO_FACE_DETECTED = 1